packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -908,15 +908,15 @@ export async function upgradeGenericHelmChart(
     )
     await installHelmDiffPlugin()
     await helmCommand(
-      `helm diff upgrade -C 5 -f ${chartDir}/values.yaml ${valuesOverride} ${releaseName} ${chartDir} --namespace ${namespace} ${parameters.join(
+      `helm diff upgrade --install -C 5 -f ${chartDir}/values.yaml ${valuesOverride} ${releaseName} ${chartDir} --namespace ${namespace} ${parameters.join(
```

Comment:
With this change (that I +1), would it make sense to merge `installGenericHelmChart` and `upgradeGenericHelmChart` into a single function?

Reply:
The rationale to include this change here is when you are upgrading `prometheus` and `grafana` is not deployed, that we can consider as a corner case (as prometheus package is composed by two different charts, which is something we may consider to refactor).

I though about changing in the past. Only concern is if we want to fail if we run `deploy install` and there is already a deployment, or when we run `deploy upgrade` and there is not a deployment. I'd check other packages to confirm if there is any impact with that change, and maybe we should create a new issue related to check/implement if we can merge install/upgrade functionality.

Reply:
Works for me, also not a needed change to merge this PR.